src/app/jobs/weatherevents.job.ts

```diff
@@ -0,0 +1,51 @@
+import { Job } from '../../common/job';
+import { IContainer } from '../../common/types';
+import { TextChannel } from 'discord.js';
+
+interface IWeather_Event {
+  features?: (IFeaturesEntity)[] | null;
+}
+
+interface IFeaturesEntity {
+  properties: IProperties;
+}
+
+interface IProperties {
+  severity: string;
+  headline: string;
+  senderName: string;
+}
+
+export class WeatherEventsJob extends Job {
+  // Run once an hour
+  public interval: number = 1000 * 60 * 60;
+
+  public name: string = 'weather_events';

```

Comment:
Put the name above the interaval, and just use normal english lol
```
'Weather Events'
```

Reply:
Will do!